From mithlesh@linsyssoft.com  Fri Oct  2 11:23:49 2009
From: Mithlesh Thukral <mithlesh@linsyssoft.com>
Date: Mon, 21 Sep 2009 11:53:27 -0700 (PDT)
Subject: Staging: comedi: drivers.c: checkpatch fix
To: Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <alpine.LFD.2.00.0909211152080.9268@localhost.localdomain>


Fix a checkpatch.pl error. Fix struct * foo to struct *foo

Signed-off-by: Mithlesh Thukral <mithlesh@linsyssoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/comedi/drivers.c
+++ b/drivers/staging/comedi/drivers.c
@@ -387,7 +387,7 @@ static int insn_rw_emulate_bits(struct c
 	return 1;
 }
 
-static inline unsigned long uvirt_to_kva(pgd_t * pgd, unsigned long adr)
+static inline unsigned long uvirt_to_kva(pgd_t *pgd, unsigned long adr)
 {
 	unsigned long ret = 0UL;
 	pmd_t *pmd;
